kitsune/urls.py

```diff
@@ -84,6 +85,9 @@
 
     # These API urls include both v1 and v2 urls.
     (r'^api/', include('kitsune.users.urls_api')),
+
+    # contribute.json url
+    url(r'^(?P<path>contribute\.json)$', servestatic, kwargs={'document_root': settings.ROOT}),
```

Comment:
We used to route this kind of url to `kitsune.sumo.urls`
[here](https://github.com/mozilla/kitsune/blob/ae1f5ff5b160d460f7c45005c1f6ec50b18ad241/kitsune/sumo/urls.py#L15)
Like we server `robots.txt` from the route. I think this url rule should live there